javascript/javascript1/week1/house estimator.js

```diff
@@ -0,0 +1,17 @@
+const Peter = 8*10*10 * 2.5 * 1000 + 100 * 300;
+const Julia = 5*11*8 * 2.5 * 1000 + 70 * 300;
+
+const peterHouseCost = 2500000;
+const juliaHouseCost = 1000000;
+
+if(Peter>peterHouseCost){
+    console.log("Peter is Paid too little")
+} else{
+    console.log("Peter is Paid too much")
+}
+
+if(Julia>juliaHouseCost){
+    console.log("Julia is Paid too little")
+} else{
+    console.log("Julia is Paid too much")
+}
```

Comment:
This would definitely get the right result, so there is nothing wrong with it. However, it is not generic, meaning that you would have to add a lot of new code, each time you want to test a new person and their numbers. So Instead, a way to solve it, would be you make it generic e.g. something like this:

```
function checkHousePrice(name, width, height, depths, gardenArea, paidPrice) {
  let realPrice = width * height * depths * 2.5 * 1000 + gardenArea * 300;
  let priceDifference = paidPrice - realPrice;
  if (priceDifference > 0) {
    console.log(`${name} has paid too much.`);
  } else {
    console.log(`${name} has paid too little`);
  }
}
checkHousePrice("Peter", 8, 10, 10, 100, 2500000);
checkHousePrice("Julia", 5, 8, 11, 70, 1000000);
```